### PR TITLE
PHP 8.1 Compatibility Fixes

### PR DESCRIPTION
**Pull request by NikhilJoshua**
## Summary

- Fix deprecated notices
- Change `FILTER_SANITIZE_STRING` to `FILTER_SANITIZE_FULL_SPECIAL_CHARS`

### CODE REVIEW

**Comment by pavanpatil1** [comment]
Hi @NikhilJoshua, verified the fix and it is working fine. No warnings are visible now related to the plugin. 

image

File: b2w-redirection.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ function rt_blogger_to_wordpress_add_option() {
 
 	wp_enqueue_style( 'rt-blogger-to-wordpress-redirection-css', ( RT_B2WR_PLUGIN_URL . 'css/b2w-redirection.css' ), array(), filemtime( RT_B2WR_PLUGIN_DIR . 'css/b2w-redirection.css' ) );
 
-	$page = filter_input( INPUT_GET, 'page', FILTER_SANITIZE_STRING );
+	$page = filter_input( INPUT_GET, 'page', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
```
Comment (vishalkakadiya) [inline b2w-redirection.php:34]: @pavanpatil1 Why we are not using `FILTER_SANITIZE_SPECIAL_CHARS` here?

Comment (NikhilJoshua) [reply b2w-redirection.php:34]: Hi @vishalkakadiya 
The official documentation recommends using `htmlspecialchars()` as `FILTER_SANITIZE_STRING` is deprecated.
`FILTER_SANITIZE_FULL_SPECIAL_CHARS` is equivalent to calling `htmlspecialchars()`

image
image

**Review by vishalkakadiya — APPROVED** [review]
LGTM!

**Pull request merged**